selendroid-server/src/main/java/io/selendroid/server/model/AndroidNativeElementFactory.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2012-2014 eBay Software Foundation and selendroid committers.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.selendroid.server.model;
+
+import android.view.View;
+import io.selendroid.server.ServerInstrumentation;
+import io.selendroid.server.android.KeySender;
+
+class AndroidNativeElementFactory implements AndroidElementFactory {
```

Comment:
DefaultAndroidNativeElementFactory? or AndroidNativeElementFactoryImpl? Not sure what selendroid naming does here.